libs/storage/Tsavorite/cs/benchmark/FixedLenYcsbBenchmark.cs

```diff
@@ -260,34 +215,33 @@ private void RunYcsbSafeContext(int thread_idx)
                 for (long idx = chunk_idx; idx < chunk_idx + YcsbConstants.kChunkSize && !done; ++idx)
                 {
                     if (idx % 512 == 0)
-                        session.CompletePending(false);
+                        bContext.CompletePending(false);
```

Comment:
does this use of wrapper affect perf numbers?

Reply:
Other way around--this actually removed a wrapper (before, bContext redirected to ClientSession)